assets/sass/scaffolding/_content.scss

```diff
@@ -111,16 +111,28 @@
    ========================================================================= */
 
 .content-box {
+    $spacingDefault: $spacingUnit / 2;
+
     background-color: white;
     border: 1px solid get-color('border', 'base');
     border-top: 4px solid get-color('brand', 'primary');
     margin-bottom: $spacingUnit;
-    padding: $spacingUnit ($spacingUnit / 2);
+    padding: $spacingUnit $spacingDefault;
 
     @include mq('medium') {
         padding: $spacingUnit;
     }
 
+    &.content-box__inner {
+        border: none;
+        margin-left: -$spacingDefault;
+        width: calc(100% + #{$spacingDefault * 2});
```

Comment:
These two lines look a bit magic – it's because they're counteracting the padding the box itself sets, which would be a pain to remove as everything else inside it needs the padding. We use a similar pattern (the `calc` usage) in a few other places.